storage/src/mmr/operation.rs

```diff
@@ -81,6 +89,22 @@ impl<K: Array, V: Array> Operation<K, V> {
         }
     }
 
+    /// Create a new floor operation that raises the floor on inactive operations to `loc`. This
+    /// operation is not supported for stores whose operations encode to less than 9 bytes.
+    pub fn floor(loc: u64) -> Self {
+        // Panic if the encoding size is too small to support this operation type.
+        assert!(Self::LEN_ENCODED >= 9);
```

Comment:
Can this be asserted at compile-time somehow I wonder? I know golang has that `init()` function...wonder if there is something similar we can do.

Reply:
Added a compile-time assertion at the end of the impl -- there might be a more concise way though?